docs/raising-events.md

```diff
@@ -34,20 +34,33 @@ robot.FellInLove += Raise.WithEmpty(); // sender will be robot, args will be Eve
 robot.FellInLove += Raise.With(sender: robot, e: someEventArgs);
 ```
 
-Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way. 
+Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way.
 
-Note that as of version 3.1.0, for the .NET Standard version of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.
+Note that as of version 3.1.0, for the .NET Standard and .NET 4.5 versions of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.

```

Comment:
Good point about .NET 4.5 version not needing `TEventArgs` to extend `EventArgs`, but we didn't release a .NET 4.5 library until 3.4! Wording is hard, but perhaps something like

> Note that for the .NET 4.5 version of the FakeItEasy library, and the .NET Standard version starting with release 3.1.0, `TEventArgs` need not extend `EventArgs` to support the above syntax. All releases of the .NET 4.0 library still require `TEventArgs` to extend `EventArgs`.